safety-architecture/tools/callgraph-tool/Makefile

```diff
@@ -58,6 +59,21 @@ install-requirements: ## Install python requirements
 	pip3 install -r requirements.txt --no-cache-dir
 	$(call target_success,$@)
 
+install-preprequisites:
```

Comment:
I think this is a bit too much for most users, but I do understand that some might like to use this kind of a target also.

If you want to have this target included, I think it should be split into at least two targets e.g. "install-apt-requirements" and "activate-venv".

Reply:
Well, the intention was to keep the things simple for the possible future users. This will be important in CONTRIBUTING todo list. Say you have a new apt or py package that needs to be installed due to some new script functionality. Instead of checking the docs each time or wondering why your build fails, you would just be following the established routine of steps:
make install-prerequisites
make test
or something similar

Reply:
I would like to keep this single target for now until we agree on the contributing pipeline.